src/prefect/cli/server.py

```diff
@@ -347,28 +368,33 @@ def config_cmd(
         --no-upgrade, -n            Flag to avoid running a database upgrade when the
                                     database spins up
         --no-ui, -u                 Flag to avoid starting the UI
+        --external-postgres, -ep    Flag to use external postgres instance
 
     \b
-        --postgres-port     TEXT    Port used to serve Postgres, defaults to '5432'
+        --postgres-port     TEXT    Port used to serve Postgres, defaults to '5432'.
+                                    Not valid for external Postgres.
```

Comment:
We should probably warn if the user sets this _and_ `--external-postgres` and the same for the other incompatible options (although I'm not sure I feel super strongly)

Reply:
Added some warnings